system_tests/speech.py

```diff
@@ -79,6 +79,21 @@ def _make_sync_request(self, content=None, source_uri=None,
                                        speech_context=['Google', 'cloud'])
         return result
 
+    def _make_async_request(self, content=None, source_uri=None,
+                            max_alternatives=None):
+        client = speech.Client(use_gax=False)
```

Comment:
OK, so should I take out these tests and add that to `.travis.yml`? I guess I'm not sure what the ask is here exactly?